run_scripts/setup_ubuntu_vm.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
I think this might be better called setup_on_ubuntu

I first thought it was setting up an ubuntu vm, rather than on an ubuntu vm.

Reply:
renamed